pkg/api/v20191231preview/openshiftcluster_example.go

```diff
@@ -9,6 +9,7 @@ import (
 
 func exampleOpenShiftCluster() *OpenShiftCluster {
 	doc := api.ExampleOpenShiftClusterDocument()
+	doc.OpenShiftCluster.Properties.WorkerProfilesStatus = nil
```

Comment:
why is this change needed?

Reply:
oh i see. it's used in the swagger example generator.

Reply:
These functions are used by swagger to generate swagger examples (/swagger/redhatopenshift/resource-manager/Microsoft.RedhatOpenShift) of each api versions that we maintain, setting this value to nil will avoid "workerProfilesStatus" filed from swagger examples files. Function **api.ExampleOpenShiftClusterDocument()** returns OC document object defined under pkg/api/openShiftCluster.go, the same is referenced by all api versions, since we want the **workerProfilesStatus** field to be added under new API (v20230701preview) only, we will have to set this to "nil" here for the APIs for which we don't want this field.